src/middleware/trailing-slash/index.test.ts

```diff
@@ -0,0 +1,75 @@
+import { Hono } from '../../hono'
+import { trimTrailingSlash, appendTrailingSlash } from '.'
+
+describe('Resolve trailing slash', () => {
+  let app: Hono
+
+  it('Trim', async () => {
+    app = new Hono({ strict: true })
+
+    app.use('*', trimTrailingSlash())
+
+    app.get('/', async (c) => {
+      return c.text('ok')
+    })
+    app.get('/the/example/endpoint/without/trailing/slash', async (c) => {
+      return c.text('ok')
+    })
+
+    let resp: Response
+
+    resp = await app.request('/')
+    expect(resp).not.toBeNull()
+    expect(resp.status).toBe(200)
+
+    resp = await app.request('/the/example/endpoint/without/trailing/slash/')
+    expect(resp).not.toBeNull()
+    expect(resp.status).toBe(301)
```

Comment:
Can you add the test to check the `location` header value?

And, you can write the same tests for all `301` responses.

Reply:
Why not?